modules/core/src/v2/coins/algo.ts

```diff
@@ -15,10 +15,11 @@ import {
   isValidSeed,
   Encoding,
 } from 'algosdk';
-import * as stellar from 'stellar-sdk';
-import * as Bluebird from 'bluebird';
+
+import { BaseCoin, BaseCoinTransactionExplanation, KeyPair } from '../baseCoin';
+import { NodeCallback } from '../types';
+
 const co = Bluebird.coroutine;
-import { KeyPair } from '../keychains';
```

Comment:
oh i saw you removed it here? Is there a reason you want to specify keychains in Basecoin again?